README.md

```diff
@@ -2,8 +2,38 @@
 
 Yet another note taking tool.
 
-## Run in dev mode
+## Build and run
+This project is build with [rust](https://www.rust-lang.org/) and [npm](https://www.npmjs.com/).
+Please see to have it installed beforehand.
 
+Additionally, running `note.rs` will create a __GUI__ and it needs a display for this.
+See to have a setup which enables this.
+
+### Install dependencies
+`note.rs` uses [tauri](https://tauri.app/) and [vite](https://vitejs.dev/).
+
+<details><summary>How to setup <b>Ubuntu 20.04</b> with tauri and vite.</summary>
```

Comment:
Please don't use HTML tags.

Reply:
To add an collapsible section html is used as described [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections)
Do you wish to renounce on the collapsible for the sake of no html in markdown?

Reply:
yes

Reply:
It is nice, that this featue is recognized by github, but I think we don't need it now. We don't have too much text by now.

Reply:
With commit d4eb70adee4fcbe0b3a848c9d816860ef81a9312 I've removed the html tags.